### PR TITLE
Add libraries description to status

### LINKED ISSUE #627: Add extension libraries information in extended status
**Description**

For debugging purposes, it would be useful to get information on what extension libraries are available when running a Taipy GUI application.

This would use the standard "status" mechanism, with all the possible information (library, elements, modules, component names...).

**Acceptance Criteria**
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Ensure any change is well documented

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
commanded by `extended_status=True`

### CODE REVIEW

**Review by FabienLelaquais — APPROVED** [review]
I think that this is just fine.
Would grouping by lib name make users more happy? I do not know.
Thx

File: src/taipy/gui/gui.py [hunk 1]
```diff
@@ -660,8 +660,32 @@ def __serve_extension(self, path: str) -> t.Any:
...
+                            for element_name, elt in lib.get_elements().items():
+                                if isinstance(elt, Element):
+                                    elt_dict = {"name": element_name}
+                                    if hasattr(elt, "_render_xhtml"):
+                                        elt_dict["render function"] = elt._render_xhtml.__code__.co_name
+                                    else:
+                                        elt_dict["react name"] = elt._get_js_name(element_name)
+                                    elts.append(elt_dict)
+            status.update({"libraries" : libraries})
+        return status
```
Comment (dinhlongviolin1) [inline src/taipy/gui/gui.py]: Is returning here redundant

**Review by dinhlongviolin1 — APPROVED** [review]
Lgtm

**Comment by FredLL-Avaiga** [comment]
> I think that this is just fine. Would grouping by lib name make users more happy? I do not know. Thx

They are

**Pull request merged**